ouroboros-consensus/src/Ouroboros/Consensus/Mempool/Init.hs

```diff
@@ -119,6 +119,8 @@ mkMempool mpEnv = Mempool
     , getTxSize      = txSize
     }
    where MempoolEnv { mpEnvStateVar = istate
+                    , mpEnvAddTxsRemoteFifo = remoteFifo
```

Comment:
Maybe we should align all of them, or none at all :grinning:

Reply:
done